tests/test_RadClass.py

```diff
@@ -78,15 +81,13 @@ def test_integration():
     # the resulting 1-hour observation should be:
     #   counts * integration / live-time
     expected = spectra * integration / test_data.livetime
-    results = np.genfromtxt('results.csv', delimiter=',')[1, 1:]
+    results = classifier.storage.to_numpy()[1]
     np.testing.assert_almost_equal(results, expected[0], decimal=2)
 
-    os.remove('results.csv')
```

Comment:
Aren’t you still writing a CSV file? Maybe you need to turn off `store_data`?

Reply:
Currently, if `store_data=False`, `RadClass.storage` is never created. This is actually addressed in #17 (since it includes a `write()` method for `RadClass` and separates the write to file from storage). I've included a partial fix here.

Reply:
Maybe we should work on getting #17 merged first (or move even move the `write()` method stuff to a new quick PR) and then rebase this after that change